app/assets/stylesheets/replies.scss

```diff
@@ -292,23 +292,17 @@ div.post-subheader { width: 100%; }
 
     ul, ol { overflow: auto; }
   }
-}
 
-/* - Tables in post-content */
-.post-content {
+  /* - Tables in post-content */
```

Comment:
Given that we've indented this thing, can we also indent the `- Lists around .post-info-box` comment to just above `ul, ol`? And maybe also put the `blockquote` thing in here too?